spec.bs

```diff
@@ -2877,6 +2888,35 @@ CORP violation report=] algorithm, as leaving it unfenced may cause a privacy le
       1. Set |url| to |config|'s [=fenced frame config/mapped url=]'s [=mapped url/value=].
 
       1. Run steps in |config|'s [=fenced frame config/on navigate callback=].
+  
+    1. If |url| is a [=url=], |navigable| is a [=fenced navigable container/fenced navigable=], and
+       <var ignore>sourceDocument</var>'s [=node navigable=] is not |navigable|:
+
+      1. Let |config| be a new [=fenced frame config=] with the following [=struct/items=]:
+
+        : [=fenced frame config/mapped url=]
+        :: a [=struct=] with the following [=struct/items=]:
+
+           : [=mapped url/value=]
+           :: |url|
+
+           : [=mapped url/visibility=]
+           :: [=visibility/transparent=]
+
+        : [=fenced frame config/effective sandboxing flags=]
+        :: a [=struct=] with the following [=struct/items=]:
+
+           : [=effective sandboxing flags/value=]
+           :: The [=fencedframetype/default fenced frame effective sandboxing flags=].
+
+           : [=effective sandboxing flags/visibility=]
+           :: [=visibility/opaque=]
+
+        : [=fenced frame config/effective enabled permissions=]
+        :: null
+
+      1. Set <var ignore>sourceSnapshotParams</var>'s [=source snapshot params/target fenced frame
```

Comment:
At this point, we're guaranteed that the target fenced frame config is null before this step, and that we're not clobbering an existing config right? Maybe add an assert?

Reply:
Done. Also added an equivalent assert for the urn:uuid case.